test/Sign.Core.Test/TestInfrastructure/TemporaryEnvironmentPath.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+namespace Sign.Core.Test
+{
+    internal sealed class TemporaryEnvironmentPath : IDisposable
+    {
+        private const string Name = "PATH";
+
+        private readonly string? _originalEnvironmentPath;
+
+        internal TemporaryEnvironmentPath(string path)
+        {
+            _originalEnvironmentPath = Environment.GetEnvironmentVariable(Name);
+
+            string paths = _originalEnvironmentPath ?? string.Empty;
+            string newPaths = string.Join(Path.PathSeparator.ToString(), paths, path);
```

Comment:
Don't you want `path` prepended to existing one? So whatever you add is found before what already exists in `PATH` if conflict exists?

Reply:
Great point.  The product itself has the same behavior.  I'll keep that behavior for now and open an issue to think about this more.